### PR TITLE
es-lint: Added eslint

### LINKED ISSUE #37: [Enhancement] ESLint
## Feature
Together with EditorConfig, a group of eslint rules will maintain a consistent style code.

I recomend the use of Pagar.me Style Code - https://github.com/pagarme/javascript-style-guide

https://github.com/sveltejs/eslint-plugin-svelte3

### PR DESCRIPTION
**Pull request by viniciusld**
Added eslint and updated some dependencies to make svelte lint work.

## Description
Added es-lint to the project and applied some rules to it, even for svelte files.

Closes #37 

## Your checklist for this pull request

:rotating_light: Please review this items for a good pull request. :four_leaf_clover:

1. My changes are well covered by **tests** and **logs**
2. I've updated the project docs (if needed)
3. I fell safe about this implementation

In a good pull request, everything above is true :relaxed:

### CODE REVIEW

**Comment by viniciusld** [comment]
@matheusvellone adicionei o style guide ao projeto e resolvi uma parte dos erros mais simples. Existem alguns errors de `if` dentro de `else` e algumas outras mais complexas de `import` que eu preferi não arriscar por enquanto :smile:

File: package.json [hunk 1]
```diff
@@ -13,7 +17,7 @@
     "rollup-plugin-terser": "4.0.4",
     "serverless": "1.49.0",
     "sirv-cli": "0.4.0",
-    "svelte": "3.0.0"
+    "svelte": "^3.2.2"
```
Comment (lucianopf) [inline package.json]:
```suggestion
    "svelte": "3.2.2"
```

File: src/containers/Index.svelte [hunk 2]
```diff
@@ -90,30 +86,18 @@
...
-      class="nes-btn collapse"
-      on:click={collapse}
-    >
-      {#if collapsed}
-        {'>'}
-      {:else}
-        {'<'}
-      {/if}
+    <button type="button" class="nes-btn collapse" on:click={collapse}>
+      {#if collapsed}{'>'}{:else}{'<'}{/if}
```
Comment (lucianopf) [inline src/containers/Index.svelte]: Eita, esse aqui é meio estranho hein! 🤔 😂

**Comment by viniciusld** [comment]
@lucianopf done! :dancing_avocado:

**Pull request merged**